pallets/stateful-storage/src/types.rs

```diff
@@ -95,19 +95,23 @@ impl<PageDataSize: Get<u32>> Page<PageDataSize> {
 				},
 			}
 		}
+		log::info!("add_buffer len: {:?}", add_buffer.len());
```

Comment:
are these needed?

Reply:
Joe probably cleaned up in #1036 but will check